src/Test.UnitTests.Sarif.Driver/Sdk/OrderedFileSpecifierTests.cs

```diff
@@ -24,7 +24,7 @@ public class OrderedFileSpecifierTestsFixture : IDisposable
         public OrderedFileSpecifierTestsFixture()
         {
             RootDirectoryRelativePath = Guid.NewGuid().ToString();
-            RootDirectory = Path.Combine(GetThisTestAssemblyFilePath(), RootDirectoryRelativePath);
+            RootDirectory = Path.Combine(Path.GetTempPath(), RootDirectoryRelativePath);
```

Comment:
Adding `TempPath = Path.GetTempPath();` would eliminate the need to call it for each test.